src/lib.rs

```diff
@@ -227,11 +227,11 @@ pub mod sync;
 mod testing;
 mod waker;
 
-#[cfg(feature = "proc_macro")]
-use proc_macro_hack::proc_macro_hack;
 #[cfg(feature = "proc_macro")]
 #[proc_macro_hack]
 pub use genawaiter_proc_macro::sync_yield_cls;
+#[cfg(feature = "proc_macro")]
+use proc_macro_hack::proc_macro_hack;
```

Comment:
I'm pretty sure the way proc-macro-hack works is to turn every thing into a `macro_rules` sort of macro? Will these have to be imported from the root anyway then?

Reply:
I briefly played with it and it looks like you're right. That's a bummer :/